### PR TITLE
osprofiler: new recipe

### PR DESCRIPTION
**Pull request by tschoonj**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/osprofiler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - pbr >=2.0
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - netaddr >=0.7.18
+    - oslo.concurrency >=3.26.0
+    - oslo.serialization >=2.18.0
+    - oslo.utils >=3.33.0
+    - prettytable >=0.7.2
```
Comment (marcelotrevisani) [inline recipes/osprofiler/meta.yaml:30]:
```suggestion
    - prettytable >=0.7.2,<0.8
```

**Comment by marcelotrevisani** [comment]
If you want to, next time you can give a go to [grayskull :skull: ](https://github.com/marcelotrevisani/grayskull/) to generate conda recipes for packages on pypi

File: recipes/osprofiler/meta.yaml (hunk 1 above)
Comment (tschoonj) [reply recipes/osprofiler/meta.yaml:30]: I am not in favour of this. The openstack people have a bad habit of adding upper version constraints in requirements.txt to exclude future major updates and then never seem to change them again, even though they work with these newer versions of the dependencies.

**Comment by tschoonj** [comment]
Thanks I will give it a try: I am using `conda skeleton pypi` now, and I need to make a lot of changes to the recipes to make them work.

Can you also have a look at https://github.com/conda-forge/cliff-feedstock/pull/21? I would like to become a maintainer of that abandoned repo.

**Comment by tschoonj** [comment]
Hi @marcelotrevisani 

Would it be possible to merge this one in?

Many thanks!

**Pull request merged**